osu.Game/Screens/Edit/Timing/DifficultySection.cs

```diff
@@ -28,7 +28,16 @@ protected override void OnControlPointChanged(ValueChangedEvent<DifficultyContro
         {
             if (point.NewValue != null)
             {
-                multiplierSlider.Current = point.NewValue.SpeedMultiplierBindable;
+                var selectedPointBindable = point.NewValue.SpeedMultiplierBindable;
+
+                // there may be legacy control points, which contain infinite precision for compatibility reasons (see LegacyDifficultyControlPoint).
+                // generally that level of precision could only be set by externally editing the .osu file, so at the point
+                // a user is looking to update this within the editor it should be safe to obliterate this additional precision.
```

Comment:
Note that this code will also run on entering the timing section in the first place, so this ends up just uniformly dropping the extra precision on all maps opened in lazer. The precision change also seems like an operation that cannot be undone (even if the intention was to apply it on first change).

Reply:
just for the current time/selection right? i breakpointed this and it didn't look to be running unexpectedly.

Reply:
Yeah you're right, it happens on first selection, not on entering the screen itself. Still seems unintended to lose precision off of a single item selection without any actual change, though.